app/views/static_pages/index.html.erb

```diff
@@ -69,30 +72,55 @@
           </li>
         <% end %>
       </ul>
+    <% end %>
 
-      <% if @events.hidden.any? %>
-        <details class="w-full text-left">
-          <summary><h4 class="inline-block pb-0 border-none muted">Hidden organizations</h4></summary>
-          <ul class="grid grid--medium-narrow text-left mt-0">
-            <%= render partial: "events/event_card", collection: @events.hidden, as: :event %>
-          </ul>
-        </details>
-      <% end %>
+    <% if @events.hidden.any? %>
+      <details class="w-full text-left">
+        <summary><h4 class="inline-block pb-0 border-none muted">Hidden organizations</h4></summary>
+        <ul class="grid grid--medium-narrow text-left mt-0">
+          <%= render partial: "events/event_card", collection: @events.hidden, as: :event %>
+        </ul>
+      </details>
+    <% end %>
 
+    <% if @events.empty? %>
+      <div class="center mt0 mb0 pt4 pb4 slate bold h3 mx-auto rounded-lg border flex flex-col w-full items-center justify-center">
+        <span>
+          No organizations associated with your account yet.
+        </span>
+        <%= link_to apply_form_url(utm_campaign: "hcb-new-event", utm_content: "first-time"), class: "btn bg-success mt2" do %>
+          <%= inline_icon "plus" %>
+          New organization
+        <% end %>
+      </div>
     <% end %>
+
     <% if @invites.any? %>
-      <h2 class="w-full text-left mt-8 pb-0 border-none">Your pending invitations</h2>
+      <h2 class="w-full text-left mt-8 pb-0 border-none h2 pl-0 ml-0">Your pending invitations</h2>
       <ul class="grid grid--narrow text-left w-full mt-2">
         <% @invites.each do |invite| %>
           <%= link_to invite do %>
-            <li class="card">
-              <%= status_badge "info" %>
-              <strong><%= invite.event.name %></strong>
+            <li class="overflow-visible card card--hover flex flex-row justify-between items-center gap-2">
+              <span>
+                <%= status_badge "info" %>
+                <strong><%= invite.event.name %></strong>
+              </span>
             </li>
           <% end %>
         <% end %>
       </ul>
     <% end %>
+      <div class="flex flex-row w-full justify-between items-center">
+        <h2 class="heading h2 line-height-4 my-2 mr-auto py-2 px-4 ml-0 pl-0 border-none">Explore HCB</h2>
```

Comment:
```suggestion
        <h2 class="heading h2 line-height-4 my-2 mr-auto py-2 px-4 ml-0 pl-0 border-none">Explore organizations running on HCB</h2>
```
Or maybe "What people are using HCB for"

Reply:
Hmm, I'm concerned this might be a bit too long on narrower screens

Reply:
Yeah I'm trying to think of a shorter version

Reply:
Explore featured organizations?